README.md

```diff
@@ -854,6 +855,139 @@ registration: {
 ```
 - **registration.click** - Function that is called when the registration button is clicked
 
+
+## Registration
+
+```javascript
+    var signIn = new OktaSignIn({
+      baseUrl: 'https://acme.okta.com',
+      clientId: '{{myClientId}}',
+      registration: {
+        parseSchema: function(schema, onSuccess, onFailure) {
+           // handle parseSchema callback
+        },
+        preSubmit: function (postData, onSuccess, onFailure) {
+           // handle preSubmit callback
+        },
+        postSubmit: function (response, onSuccess, onFailure) {
+            // handle postsubmit callback
+        }
+      },
+      features: {
+         registration: true
+      }
+    });
+```
+
+Following configurations are needed to enable registration on a self hosted sign in widget
```

Comment:
We should be able to remove this line and add inline comments in the code block above.

```javascript
var signIn = new OktaSignIn({
      baseUrl: 'https://acme.okta.com',
      clientId: '{{myClientId}}',  // REQUIRED
      registration: {
        parseSchema: function(schema, onSuccess, onFailure) {
           // handle parseSchema callback
        },
        preSubmit: function (postData, onSuccess, onFailure) {
           // handle preSubmit callback
        },
        postSubmit: function (response, onSuccess, onFailure) {
            // handle postsubmit callback
        }
      },
      features: {
         registration: true  // REQUIRED
      }
    });
```

Reply:
Fixed